settings.gradle.kts

```diff
@@ -0,0 +1,8 @@
+rootProject.name = "HelloWorldExtension"
+rootProject.buildFileName = "build.gradle.kts"
```

Comment:
```suggestion
```
This is unnecessary, as gradle will look for `build.gradle` and `build.gradle.kts` files by default